### PR TITLE
Move CSS to separate stylesheet

### PR DESCRIPTION
**Pull request by speakerbug**
This moves the CSS to a separate stylesheet. It also updates the CSS for 'black' and 'red' to be their respective hex values.

### CODE REVIEW

**Comment by speakerbug** [comment]
Sorry, there were a bunch of commits of me just trying to get my forked repo up to date with this branch's master. Feel free to just squash all of them.

File: server-frontend/src/server-frontend/client/style.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+th {
+    font-weight: bold;
+    font-size: larger;
+}
+
+.center {
+    text-align: center;
+}
```
Comment (tadhunt) [inline server-frontend/src/server-frontend/client/style.css]: Please fix missing NL at EOF

File: server-frontend/src/server-frontend/client/style.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+.game-chat {
+    margin-top: 10px;
+    padding-top: 10px;
+    color: #FCFB78;
+}
+
+body {
+    background: #000;
```
Comment (tadhunt) [inline server-frontend/src/server-frontend/client/style.css]: Please s/#000/#000000/

**Review by tadhunt — CHANGES_REQUESTED** [review]
Please make the two minor changes commented in this review.  Additionally I'm about to send you a PR against this branch for the changes necessary for this change to work in App Engine.  (Need to teach App Engine where to find style.css)

File: server-frontend/src/server-frontend/client/style.css (hunk 1 above)
Comment (speakerbug) [reply server-frontend/src/server-frontend/client/style.css]: I always forget about the NL...

**Comment by speakerbug** [comment]
Okay, those changes should be made now. I'm not exactly sure why the NL at EOF isn't showing up on the comment above since it should be in there now...let me know if you can't see it.

**Review by tadhunt — APPROVED** [review]
LGTM

**Pull request merged**